rfcs/xxxx-rfc-process/text.md

```diff
@@ -0,0 +1,60 @@
+
+**Reviewers:** [Madison Dunitz](madison.dunitz@chanzuckerberg.com), [Brian McCandless](bmccandless@chanzuckerberg.com), [Trent Smith](trent.smith@chanzuckerberg.com), [Colin Megill](colin.megill@chanzuckerberg.com), [Seve Badajoz](sbadajoz@chanzuckerberg.com)
+
+## tl;dr 
+
+This RFC contains the details about the RFC process for all cellxgene related designs.
+
+
+## Background
+
+Previously the "RFC" (a.k.a. "design doc review") process was executed on Google Docs. There were two problems
+
+1. Updates were quite tricky to make. Do you update the doc directly? Do you have a changelog? Which version is latest? Do you need to apply the change in the changelog in the body of the doc? 
+2. Designs decisions were not easily accessible by our external community. How does the external community, with whom we have a close relationship, keep track of our decisions and progress if the designs are hidden in a private Google Drive?
+
+In order to alleviate these questions and concerns, we have moved to Github to keep track of our RFCs moving forward.
+
+
+## How to create a new RFC
+
+The following are the steps to writing and approving a new RFC (not modifying):
+
+1. Create a new branch of the `cellxgene-rfcs` repo.
+2. Create a new directory under the `rfcs` directory named `XXXX-name-of-rfc`, where `name-of-rfc` is the name of your RFC, not those words directly.
+3. Copy over the contents of the `templates` directory into your newly created directory. This contains a folder for images and the RFC template.
+4. Rename `rfc_template.md` to `text.md` and edit it. Add images that you'd like to include in the `imgs` folder.
+5. When you are ready for review, create a pull request. Add your approvers (and any other people you'd like to review your work) as reviewers on the PR. Please note that [Girish Patangay](mailto:girish.patangay@chanzuckerberg.com) and [Sara Gerber](mailto:sara.gerber@chanzuckerberg.com) are required Reviewers for all RFCs so that CZI Central Infra and CZI Security teams are aware of cellxgene technical work. They may be removed as reviewers if the RFC is non-technical. 
+6. As a courtesy, post a heads up on the [#org-sci-eng-single-cell](https://chanzuckerbergteam.slack.com/archives/GQGPP7925) Slack channel that there is a design available for review, specifically calling out the approvers. Generally, two weeks is nice for substantial designs to get reviewed, but use your best judgement on how much time you'd like for your design to be under review.
+7. Once all comments are addressed and the *approvers* have approved your pull request, **change the name of your directory** from `XXXX-name-of-rfc` to pick the next number (for example, this one will be `0000-rfc-process`).
+8. Merge your PR! Delete your branch! Close your design ticket! Code away!
+
+
+## How to update an existing RFC
```

Comment:
Generally editorial and non-normative changes do not require another review cycle? For example, _Minor and trivial changes MAY be made freely, with no announcement by the author(s) of the RFCs. Such changes include correcting typos, re-wordings, and the like._

Reply:
Thank you! I've copied over your wording and lowercased the "MAY" as I've opted this time to make the language a little less rigorous.